apps/modernization-ui/src/apps/page-builder/page/management/edit/staticelement/AddStaticElement.spec.tsx

```diff
@@ -12,12 +12,16 @@ const page: PagesResponse = {
     status: 'Draft'
 };
 
+const fetch = () => {
+    console.log("dummy fetch");
+}
```

Comment:
Suggestion: Can mock functions using `jest.fn()`. Can then assert that they are called when expected